### PR TITLE
Add home page

### PR DESCRIPTION
**Pull request by Haider8**
News items are yet to be shown on the home page.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,6 +26,7 @@
     "@vue/cli-service": "^3.3.0",
     "@vue/eslint-config-prettier": "^4.0.1",
     "lint-staged": "^8.1.0",
+    "material-design-icons-iconfont": "^4.0.4",
```
Comment (iamareebjamal) [inline package.json]: This is not needed, we already have material design icons set up

File: src/plugins/vuetify.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import Vue from 'vue'
 import Vuetify from 'vuetify'
 import 'vuetify/dist/vuetify.min.css'
+import 'material-design-icons-iconfont/dist/material-design-icons.css' // Ensure you are using css-loader
```
Comment (iamareebjamal) [inline src/plugins/vuetify.js]: Unneeded

**Comment by iamareebjamal** [comment]
@Haider8 There are 11 commits, there should only be one. When creating new PRs, please checkout to master branch, pull new changes from upstream and create new branch from there

File: package.json (hunk 1 above)
Comment (Haider8) [reply package.json]: But it was not working for me

Comment (iamareebjamal) [reply package.json]: Try pulling new changes from master and then adding these commits on top of it. I added those in one of your PR, that change is not in your branch

**Comment by Haider8** [comment]
@iamareebjamal So i have to make new PR now?

**Comment by iamareebjamal** [comment]
No, you can rebase from master in this branch and force push the new changes

**Comment by Haider8** [comment]
Build failed in installing the new changes after rebasing from master

**Comment by iamareebjamal** [comment]
Good job. But it looks squished on smaller screens. Go in responsive mode and resize the width of the page

**Pull request merged**